CMakeLists.txt

```diff
@@ -304,6 +304,9 @@ OCV_OPTION(WITH_GTK "Include GTK support" ON
 OCV_OPTION(WITH_GTK_2_X "Use GTK version 2" OFF
   VISIBLE_IF UNIX AND NOT APPLE AND NOT ANDROID
   VERIFY HAVE_GTK AND NOT HAVE_GTK3)
+OCV_OPTION(WITH_FRAMEBUFFER "Include framebuffer support" ON
+  VISIBLE_IF UNIX AND NOT APPLE AND NOT ANDROID
+  VERIFY HAVE_FRAMEBUFFER)
```

Comment:
Please remove `VERIFY HAVE_FRAMEBUFFER` part, it does not work and probably is useless anyway.

Reply:
done